src/front-end/index.html

```diff
@@ -65,19 +67,28 @@
   </div>
 
   <button id="pay-button" disabled="">
+    <!-- <span id="refresh-icon"><img src="./images/refresh_white.png" alt="refresh"></span> -->
     <div id="button-message">Pay Now</div>
     <div id="button-dots" class="dots white-dots">
       <span></span>
       <span></span>
       <span></span>
     </div>
+
+    <div class="checkmark draw"></div>
+
+    <!-- Cross -->
+    <svg class="container container--fail" viewBox="0 0 50 50">
```

Comment:
Does the SVG have to be referenced like this or can it be a separate file with the SVG?

Reply:
I could put it in a separate file and embed in the HTML within an object, I'd then need to move the CSS to that SVG file also. Not sure how I would reference it in my javascript to change display properties though...